baseplate/lib/prometheus_metrics.py

```diff
@@ -88,3 +89,106 @@ def get_requests_total_metric(self) -> Counter:
     def get_active_requests_metric(self) -> Gauge:
         """Return the active_requests metrics"""
         return thrift_server_active_requests
+
+
+# http server labels and metrics
+http_server_histogram_labels = [
+    "http_method",
+    "http_endpoint",
+    "http_success",
+]
+http_server_requests_total_labels = [
+    "http_method",
+    "http_endpoint",
+    "http_success",
+    "http_response_code",
+]
+http_server_active_requests_labels = [
+    "http_method",
+    "http_endpoint",
+]
+
+http_server_latency_seconds = Histogram(
+    "http_server_latency_seconds",
+    "Description of histogram",
```

Comment:
Should this be a description of this histogram?

Reply:
It should indeed :sweat_smile: